structuredefinitions/Extension-England-RemovalReason.xml

```diff
@@ -0,0 +1,61 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-RemovalReason" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-RemovalReason" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandRemovalReason" />
+  <title value="Extension England Removal Reason" />
+  <status value="draft" />
+  <date value="2023-07-19" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="An Extension used for the removal of an FGM Flag" />
+  <purpose value="The Extension extends the Flag resource to remove the FGM Flag if it's no longer in use" />
+  <copyright value="Copyright © 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the \&quot;License\&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \&quot;AS IS\&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="rim" />
+    <uri value="http://hl7.org/v3" />1
+    <name value="RIM Mapping" /></mapping>
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="Flag" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension.url">
+      <path value="Extension.url" />
+      <fixedUri value="https://fhir.nhs.uk/StructureDefinition/Extension-England-RemovalReason" />
```

Comment:
Need to update to https://fhir.nhs.uk/StructureDefinition/Extension-England-FGMRemovalReason